frontend_lib/src/component/EmojiReactions/EmojiReactionButton.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import { translate } from 'react-i18next'
+import { humanAndList } from '../../helper.js'
+import { getEmojiDataFromNative } from 'emoji-mart'
+import emojiData from 'emoji-mart/data/all.json'
+import Radium from 'radium'
+
+const color = require('color')
+
+const primaryColor = GLOBAL_primaryColor // eslint-disable-line camelcase
+const BUTTON_STYLE = {
```

Comment:
There is a reason to use this instead of the `.primaryColorBorder` and `.primaryColorBgLighten` classes?

Reply:
Okay, I've changed the code to use `primaryColorBorder`. `primaryColorBgLighten` is however way too dark and seems to make the emoji difficult to read. We can discuss to find another solution if you want.

Reply:
if it is too dark, you can leave it as it was before (or add opacity to the background in the "highlighted" class), my question was just to know why it hadn't been used